File: src/chrome/android/java/src_wooeen_base/com/wooeen/model/to/VersionTO.java
package com.wooeen.model.to;

public class VersionTO {

    private int checkout;
    private int product;
    private int query;
    private int pub;
    private String android;

    public int getQuery() {
        return query;
    }

    public void setQuery(int query) {
        this.query = query;
    }

    public int getProduct() {
        return product;
    }

    public void setProduct(int product) {
        this.product = product;
    }

    public int getCheckout() {
        return checkout;
    }

    public void setCheckout(int checkout) {
        this.checkout = checkout;
    }

    public int getPub() {
        return pub;
    }

    public void setPub(int pub) {
        this.pub = pub;
    }

    public String getAndroid(){
        return android;
    }

    public void setAndroid(String android){
        this.android = android;
    }
}

File: src/brave/ios/brave-ios/Sources/Wooeen/Model/VersionTO.swift
import Foundation

public struct VersionTO: Decodable {
    public var checkout: Int?
    public var product: Int?
    public var query: Int?
    public var pub: Int?
    public var android: String?
    public var ios: String?
}

public struct VersionResponse: Decodable{
    public var result: Bool
    public var callback: VersionTO?
    public var message: String?
}
